sql-gremlin/src/main/java/org/twilmes/sql/gremlin/adapter/converter/ast/nodes/select/GremlinSqlSelectSingle.java

```diff
@@ -233,6 +242,23 @@ private void appendByGraphTraversal(final GremlinSqlNode gremlinSqlNode, final S
             } else {
                 graphTraversal.by(graphTraversal1);
             }
+        } else if (gremlinSqlNode instanceof GremlinSqlLiteral) {
+            final GremlinSqlLiteral gremlinSqlLiteral = (GremlinSqlLiteral) gremlinSqlNode;
+            final List<SqlNode> sqlNodeList = sqlSelect.getSelectList().getList();
+            if (gremlinSqlLiteral.getValue() instanceof Number) {
+                final Number value = (Number) gremlinSqlLiteral.getValue();
+                if (sqlNodeList.size() <= value.intValue() || value.intValue() <= 0) {
+                    appendByGraphTraversal(GremlinSqlFactory.createNode(sqlNodeList.get(value.intValue() - 1)), table,
+                            graphTraversal);
+                } else {
+                    throw new SQLException("Error, ordinal value of ORDER BY must be between 1 and the number of columns (inclusive).");
+                }
+            } else {
+                throw new SQLException("Error, invalid ORDER BY column literal.");
+            }
+        } else {
+            throw new SQLException(
+                    String.format("Error, could not apply ORDER BY to {}.", gremlinSqlNode.getClass().getName()));
```

Comment:
Do we have options to elaborate on the error message? In other words is it something that user did wrong and should fix, or possibly an unsupported feature. If it is the last scenario - we should indicate user can check out the page with the list of limitations and supported features.